tox.ini

```diff
@@ -24,6 +24,10 @@ skipsdist = true
 # repository anyway)
 commands =
     pip install ./python/
+    # pin python-xlib to version 0.31 until the issue
+    # https://github.com/python-xlib/python-xlib/pull/242
+    # is fixed.
+    pip install --force-reinstall -v "python-xlib==0.31"
```

Comment:
Is it also possible to require the version in the `deps` line directly?

```
deps =
    [...]
    python-xlib==0.31
```

Reply:
Indeed, this seems to work too! I didn't dare to try it because I couldn't find an example online.

Reply:
Nice :partying_face: